ui/tests/integration/components/secret-engine/configuration-details-test.js

```diff
@@ -48,16 +48,29 @@ module('Integration | Component | SecretEngine/ConfigurationDetails', function (
       );
 
       for (const key of expectedConfigKeys(type)) {
```

Comment:
this test changed because I made my helper `expectedConfigKeys` a little easier to read. Instead of creating config arrays the excluded the keys not returned from the api, I just exclude them in the loop and check they're not in the view.